### ISSUE TITLE
[Linux compilation] Python2 EOL; Python3 unsupported

### ISSUE THREAD

**Issue by KAMiKAZOW** [yuzu-emu/yuzu#3066]
```
[  322s] Scanning dependencies of target unicorn-build
[  322s] [  0%] Generating ../externals/unicorn/libunicorn.a
[  322s] 
[  322s] ERROR: Cannot use '/usr/bin/python', Python 2.4 or later is required.
[  322s]        Note that Python 3 or later is not yet supported.
[  322s]        Use --python=/path/to/python to specify a supported Python.

```

All Linux distributions are in the process of removing Python2 or have already done so, including but not limited to the current Debian development branch which the upcoming Ubuntu LTS release will be based on.

**Comment by lioncash** [comment]
Can you try updating the submodule to track the latest commit and report back with whether or not it fails. It should be compatible with Python 3.

**Comment by KAMiKAZOW** [comment]
No, I can't. Sorry.

**Comment by lioncash** [comment]
We don't use mainline unicorn since it doesn't contain all the features we need.

**Comment by jbeich** [comment]
yuzu-emu/unicorn master [does help](https://github.com/yuzu-emu/yuzu/files/4001017/yuzu-qt5-s20191225.log) and [so does](https://github.com/yuzu-emu/yuzu/files/4001025/yuzu-qt5-s20191225.log) backporting https://github.com/unicorn-engine/unicorn/pull/1145. Tested with Python 3.7.

**Comment by KAMiKAZOW** [comment]
So JBeich conformed it working last year. What's the holdup in updating the submodule?

**Comment by Morph1984** [comment]
Closed by #4888

**Comment by baryluk** [comment]
I still see references to `python2` and `python` in build instructions here https://github.com/yuzu-emu/yuzu/wiki/Building-For-Linux
